book.toml

```diff
@@ -0,0 +1,16 @@
+[book]
+authors = ["Ream Contributors"]
+language = "en"
+multilingual = false
+src = "book"
+title = "ream Book"
+
+[build]
+build-dir = "target/book"
+
+[output.html]
+no-section-label = true
+
+[output.html.fold]
+enable = true
+level = 1
```

Comment:
add a new line to this final to remove the 
![image](https://github.com/user-attachments/assets/015562db-b440-4a94-8a81-d6c62428016d)